shaders/src/main/glsl/samples/320es/stable_mergesort.frag

```diff
@@ -83,50 +117,35 @@ void mergeSort()
 
 void main()
 {
+ // Note the use of injectionSwitch uniform to avoid compiler
+ // optimization of the population of initial data[] values. 
```

Comment:
I would remind the reader that injectionSwitch has the value (0.0, 1.0)

Reply:
Done